src/docs/asciidoc/tcp-client.adoc

```diff
@@ -439,3 +476,79 @@ public class Application {
 }
 ----
 ====
+
+== Metrics
+TCP client supports built-in integration with https://micrometer.io/[`Micrometer`].
+It exposes all metrics with prefix `reactor.netty.tcp.client`.
+
+The following table provides information for the TCP client metrics:
+
+[width="100%",options="header"]
+|=======
+| metric name | type | description
+| reactor.netty.tcp.client.data.received | DistributionSummary | Amount of the data that is received, in bytes
```

Comment:
Remove "that is" in all of these.

(Same comment as before, that you can be more telegraphic in tables.)